### DISCUSSION TITLE
Table spanning multiple pages

### DISCUSSION THREAD

**Discussion by BenoitNivelles** [jsvine/pdfplumber#1152] [Q&A]
Hello. I need to extract a table spanning multiple pages.  The header page[0] is repeated on each page. I can python code a specific logic, but I m wondering if a more 'plumber' lovic would exist. Thanks for your advice.

**Comment by jsvine** [comment]
Hi @BenoitNivelles, and thanks for your interest in `pdfplumber`. Currently, there's not a more `pdfplumber`-esque approach than writing your own custom logic. But definitely something worth me considering as a future feature.
